src/plugins/data-card/components/case-attribute.tsx

```diff
@@ -344,6 +341,17 @@ export const CaseAttribute: React.FC<IProps> = observer(props => {
     return <span></span>; // There may be more cases in the future, e.g. date picker
   };
 
+  const itemWithBoldedMatch = (fullString: string, matchString: string) => {
+    // If full string is "Orange" and matchString is "ran"
+    // the result will be "O<b>ran</b>ge"
+    const matchIndex = fullString.toLowerCase().indexOf(matchString.toLowerCase());
+    const matchEndIndex = matchIndex + matchString.length;
+    const match = fullString.slice(matchIndex, matchEndIndex);
+    const lettersBeforeMatch = fullString.slice(0, matchIndex);
+    const lettersAfterMatch = fullString.slice(matchEndIndex);
+    return <span>{lettersBeforeMatch}<b>{match}</b>{lettersAfterMatch}</span>;
```

Comment:
In the case of an empty `matchString`, this will return `<span><b></b>{fullString}</span>` which may look ok to the user but seems somewhat inelegant and potentially confusing -- it took me some MDN-sleuthing to figure it out. Seems like the code would be clearer with an opt-out at the top of the form:
```typescript
    if (!matchString) return <span>{fullString}</span>;
```